README.md

```diff
@@ -454,6 +454,12 @@ object containing the following properties:
   (#authenticating). If defined, this overrides the option of the same name defined on the entire
   route.
 
+##### `authOptional`
+- _String_
+- Default: `undefined`
+- If true, this endpoint will optionally require login auth, if user is logged in, then endpoint 'user' & 'userId' exists. If not logged in / auth error, endpoint '.user' & 'userId' will be undefined. User is still accessible to the API but as a guest then. If `authRequired` is true, this setting will be overridden.
+
```

Comment:
Make sure to highlight any code using the proper markdown syntax. Also, those values are available at `this.user` and `this.userId`. And I would change "If not logged in or there's an auth error" to "If not logged in / auth error". Here. Let me save you some time. Change this and the one below to:

>If true, authentication is optional at this endpoint. If the request is authenticated, `this.user` & `this.userId` will be available in the [endpoint context](https://github.com/kahmali/meteor-restivus#endpoint-context). If authentication fails, the endpoint will still run, but `this.user` & `this.userId` will be `undefined`. If `authRequired` is `true`, this setting will be overridden and have no effect.